posts/7-common-errors-to-check-when-debugging-airflow-dag.md

```diff
@@ -1,223 +1,203 @@
 
-**Note:** Airflow v1.10.2's new sensor [`mode=reschedule`](https://github.com/apache/airflow/blob/1.10.2/airflow/sensors/base_sensor_operator.py#L46-L56) feature addresses this issue. If you have more sensors than worker slots, the sensor will now get thrown into a new `up_for_reschedule` state, thus unblocking a worker slot.
+> **Note:** Airflow v1.10.2's new sensor [`mode=reschedule`](https://github.com/apache/airflow/blob/1.10.2/airflow/sensors/base_sensor_operator.py#L46-L56) feature addresses this issue. If you have more sensors than worker slots, the sensor will now get thrown into a new `up_for_reschedule` state, thus unblocking a worker slot.
+>
+> If you're running Airflow 2.0+, consider [Smart Sensors](https://airflow.apache.org/docs/apache-airflow/2.0.1/smart-sensor.html?highlight=smart%20sensors).
 
-### 5. Tasks are executing, but they're getting bottlenecked
+## 5. Tasks are executing, but they're getting bottlenecked.
 
-If everything looks like it's running as expected but you're finding that your tasks are getting bottlenecked, we'd recommend taking a closer look at two things:
+If everything looks like it's running as expected but you're finding that your tasks are getting bottlenecked, we'd recommend taking a closer look at:
 
-Your Env Vars & Concurrency Related Configs + your Worker and Scheduler Resources.
+- Environment Variables and concurrency configurations
+- Worker and Scheduler resources
 
+### Environment Variables and concurrency configurations.
 
-#### 1. Check your Env Vars & Concurrency Related Configs
+The potential root cause for a bottleneck and what exactlhy these values should be set at is specific to your setup. For example, are you running many DAGs at once, or one DAG with hundreds of concurrent tasks?
 
-What exactly these values should be set at (and what could be causing a potential bottleneck) is specific to your setup - e.g. are you running many DAGs at once or one DAG with hundreds of concurrent tasks? With that said, fine-tuning them can certainly help address performance issues. Here's a breakdown of what you can look for:
+Regardless of your use case, however, fine-tuning some configurations can help improve performance. Here's a breakdown of what you can look for:
 
-** 1. Parallelism** ([parallelism](https://github.com/apache/airflow/blob/v1-10-stable/airflow/config_templates/default_airflow.cfg#L113))
-
-   *  This determines how many task instances can be actively running in parallel _across_ DAGs given the resources available at any given time at the deployment level. Think of this as "maximum active tasks anywhere."
+**a. Parallelism** ([parallelism](https://github.com/apache/airflow/blob/v1-10-stable/airflow/config_templates/default_airflow.cfg#L113))
+   *  This determines how many task instances can be actively running in parallel _across_ DAGs given the resources available at any given time at the Deployment level. Think of this as "maximum active tasks anywhere."
```

Comment:
```suggestion
   The `AIRFLOW__CORE__PARALLELISM` environment variable determines how many task instances can be actively running in parallel across DAGs given the resources available at any given time at the Deployment level. Think of this as "maximum active tasks anywhere." To increase performance, set this value higher than its default value of 32.
```
This is an example of what I'd like this section to look like. Feel free to implement something similar across the other variables, or save it for later.